src/System.Device.Gpio/System/Device/Gpio/Drivers/UnixDriver.Windows.cs

```diff
@@ -1,84 +1,84 @@
-﻿// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
Same goes for this, I guess they are pretty much the same right? If it was EOL sequence then I do want to keep your changes as I noticed that they were wrong but haven't fixed them yet.

Reply:
Yes, EOL related.